balancer-labs/frontend-v2
# Pool creation unknown tokens change price

# Description

After some discussion with Pon, we came up with a solution that should also allow the user to change any token prices that they may have entered. This PR introduces a small 'Token Prices' card under Pool Summary which allows the user to go back and change any token prices they might've mistakenly, or had forgotten to enter.

The modal appearance behaviour has also changed. Instead of appearing immediately after selecting an unknown token, it will now appear on the next step as per design comments.


## Type of change


## How should this be tested?
Here is a custom token: 0xbF3122A7a98a2F9971C339a8511378d853585876
Using an unknown token, add the  unknown token. It should appear on the side card with a warning to enter a price. 
You should be able to click the red 'Enter a price' row and it should bring up the unknown token price modal where you will be able to enter in a price. Test that this works properly

Another one to check is to make sure that the modal shows up on the next step (step 2: select fees) and applies the prices correctly there too.

Also please test mobile accordion behaviour.

## Checklist:


### Walkthrough
(Note: I have no idea why USDC is returning 0)
https://www.loom.com/share/c1bcf8e807d04b2395fa2548e0ae1713
